pkg/model/types.go

```diff
@@ -45,6 +45,10 @@ type Tensor struct {
 	Name  string `json:"name,omitempty" yaml:"name,omitempty"`
 	Size  []int  `json:"size,omitempty" yaml:"size,omitempty"`
 	DType string `json:"dtype,omitempty" yaml:"dtype,omitempty"`
+	// OpType is special for PMML
+	OpType string `json:"optype,omitempty" yaml:"optype,omitempty"`
```

Comment:
Should we define all the string to `*string` since it is optional?

Reply:
Why optional field is must be `*string`? I think whether it is `string` or `*string`,  the `omitempty` is effective.

Reply:
It's more like a syntactic sugar, maybe.

If we use *string, we can check if the value is set using `if opType == nil`.

Just one nit, doesn't matter.

/lgtm